Makefile

```diff
@@ -67,3 +67,9 @@ ifeq (, $(shell which gosec))
 	}
 endif
 	cd $(CURDIR); gosec ./...
+
+.PHONY: scan
+scan: 
+	go install golang.org/x/vuln/cmd/govulncheck@latest ;\
```

Comment:
We should check if it exists before we install ig?

Reply:
Yes, very good catch 😃. @sheharyaar can you please add a check using `command -v govulncheck` before invoking `go install`

Reply:
Sure